dydx3/modules/private.py

```diff
@@ -770,9 +792,18 @@ def sign(
         iso_timestamp,
         data,
     ):
-        return SignableApiRequest(
-            iso_timestamp=iso_timestamp,
-            method=method,
-            request_path=request_path,
-            body=json_stringify(data) if data else '',
-        ).sign(self.api_private_key)
+        message_string = (
+            iso_timestamp +
+            method +
+            request_path +
+            (json_stringify(data) if data else '')
+        )
+
+        hashed = hmac.new(
+            base64.urlsafe_b64decode(
+                (self.api_key_credentials['secret']).encode('utf-8')
+            ),
+            msg=message_string.encode('utf-8'),
+            digestmod=hashlib.sha256,
+        )
+        return base64.urlsafe_b64encode(hashed.digest()).decode()
```

Comment:
Doesn't really matter but I think would be more consistent with the other client to simply use `b64encode` instead of the url safe version

Reply:
that doesn't work based on how python base64encode works